systemtests/utils/utils.go

```diff
@@ -154,6 +153,8 @@ func StartNetPlugin(t *testing.T, nodes []TestbedNode, nativeInteg bool) {
 
 // StartNetmasterWithFlags starts netplugin on specified testbed nodes with specified flags
 func StartNetmasterWithFlags(t *testing.T, node TestbedNode, flags map[string]string) {
+	time.Sleep(5 * time.Second)
```

Comment:
IKR? It sucks.

This is however needed, because the services that netmaster rely on (consul and etcd) may not have fully coordinated/booted and thus be unavailable. I saw this happen numerous times.